lib/orders.rb

```diff
@@ -0,0 +1,41 @@
+require "order"
+
+class Orders
+  attr_reader :orders
+
+  def initialize
+    @orders = []
+  end
+
+  def add_order(order)
+    @orders << order
+  end
+
+  def get_processing_orders
+    return get_orders_by_status("PROCESSING")
+  end
+
+  def get_processed_orders
+    return get_orders_by_status("PROCESSED")
+  end
+
+  def get_orders_by_status(status)
```

Comment:
If there is function `get_orders_by_status` do we need separate `get_processing_orders` and `get_processed_orders`? Whenever this is called, we could just call `get_orders_by_status` directly.

Reply:
We could, however I found that it reads poorly in the other methods. My intent is get_orders_by_status to be a private method so no one outside of the orders class can access it.